recipe_scrapers/projectgezond.py

```diff
@@ -10,83 +8,3 @@ def host(cls):
 
     def author(self):
         return "Project Gezond"
-
-    def title(self):
-        return self.soup.find("h1", {"class": "entry-title"}).text
-
-    def category(self):
-        return ", ".join(
-            [
-                element.text
-                for element in self.soup.find(
-                    "span", {"class": "meta-category"}
-                ).find_all(
-                    "a", {"class": lambda x: x is not None and x.startswith("category")}
-                )
-            ]
-        )
-
-    def total_time(self):
-        time_element = self.soup.find("em", string="Bereidingstijd:").parent
-        return "".join(
-            [
-                element.text
-                for element in time_element.children
-                if element.text != "Bereidingstijd:"
-            ]
-        ).strip()
-
-    def yields(self):
-        # Match everything in the h2 with 'Dit heb je nodig'
-        # The text inside the parentheses contains the yield for the ingredients that are listed
-        return re.search(
-            r"Dit heb je nodig \(([^)]+)",
-            self.soup.find(
-                "h2", string=lambda x: x.startswith("Dit heb je nodig")
-            ).text,
-        ).group(1)
-
-    def ingredients(self):
-        ingredients_table = self.soup.find(
-            "h2", string=lambda x: x.startswith("Dit heb je nodig")
-        ).next_sibling.next_sibling
-        ingredients = [
-            ingredient.text
-            for ingredient in ingredients_table
-            if ingredient.text.strip()
-        ]
-        return ingredients
-
-    def instructions(self):
-        instructions_table = self.soup.find(
-            "h2", string=lambda x: x.startswith("Zo maak je het")
-        ).next_sibling.next_sibling.next_sibling.next_sibling
-        instructions = [
-            instruction.text
-            for instruction in instructions_table
-            if instruction.text.strip()
-        ]
-        return "\n".join(instructions).strip()
-
-    def ratings(self):
-        # Ratings do not exist on this site
-        return None
-
-    def cuisine(self):
-        # Not listed on site
-        return None
```

Comment:
Let's keep these two empty-result methods, unless we believe the site has begun publishing this info.

Reply:
Hmm.  Perhaps these should rase `FieldNotProvidedByWebsiteException` instead..